cg/services/order_validation_service/workflows/pacbio_long_read/validation_service.py

```diff
@@ -0,0 +1,48 @@
+from cg.services.order_validation_service.errors.order_errors import OrderError
+from cg.services.order_validation_service.errors.sample_errors import SampleError
+from cg.services.order_validation_service.errors.validation_errors import ValidationErrors
+from cg.services.order_validation_service.model_validator.model_validator import ModelValidator
+from cg.services.order_validation_service.order_validation_service import OrderValidationService
+from cg.services.order_validation_service.response_mapper import create_order_validation_response
+from cg.services.order_validation_service.utils import (
+    apply_order_validation,
+    apply_sample_validation,
+)
+from cg.services.order_validation_service.workflows.order_validation_rules import ORDER_RULES
+from cg.services.order_validation_service.workflows.pacbio_long_read.models.order import PacbioOrder
+from cg.services.order_validation_service.workflows.pacbio_long_read.validation_rules import (
+    SAMPLE_RULES,
+)
+from cg.store.store import Store
+
+
+class PacbioLongReadValidationService(OrderValidationService):
```

Comment:
too long for a name service? should it be only `PacbioValidationService`?

Reply:
I think it is "ok", either works!

Reply:
Either way works for me. If the order type is called PacbioLongRead it might be nice to have it like this for now.